ExpenseTrackerGrupo2.csproj

```diff
@@ -8,9 +8,17 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.1.35" />
+    <PackageReference Include="DotNetEnv" Version="3.1.1" />
```

Comment:
May you tell me why are you adding some new dependencies to the project?

Reply:
I was trying another solution, for what alex already did, but I will delete it